database/wiiu/get_user_friend_list.go

```diff
@@ -52,8 +55,21 @@ func GetUserFriendList(pid uint32) []*friends_wiiu_types.FriendInfo {
 		} else {
 			// Offline
 
+			userData, err := globals.GetUserData(friendPID)
+			if err != nil {
+				globals.Logger.Critical(err.Error())
+				continue
+			}
+
+			userInfo, err := GetUserInfoByPNIDData(userData)
+			if err != nil {
+				globals.Logger.Critical(err.Error())
+				continue
+			}
```

Comment:
I know this will create several duplicate review comments but I'm mostly doing this to bring attention to all the areas, and to tag @hauntii for them all